connectors/src/api/webhooks/webhook_slack.ts

```diff
@@ -181,6 +182,13 @@ const _webhookSlackAPIHandler = async (
 
     switch (event?.type) {
       case "app_mention": {
+        const slackMessage = event?.text;
+        if (slackMessage && slackMessage.trim() === "!watch") {
```

Comment:
Nice Idea!

Reply:
We would need a whitelist through a config to poke that hole safely maybe ?

Reply:
Yeah I actually wanted to ask you: do we want to only allow admins to do it?

Reply:
Yes only admins can modify what Dust can see